lib/ShiftRegister74HC595_NonTemplate/src/ShiftRegister74HC595_NonTemplate.h

```diff
@@ -0,0 +1,40 @@
+/*
+   ShiftRegister74HC595.h - Library for simplified control of 74HC595 shift registers.
+   Developed and maintained by Timo Denk and contributers, since Nov 2014.
+   Additional information is available at https://timodenk.com/blog/shift-register-arduino-library/
+   Released into the public domain.
+ */
+
+#pragma once
+
+#include <Arduino.h>
+#include <vector>
+
+class ShiftRegister74HC595_NonTemplate {
+public:
+
+  ShiftRegister74HC595_NonTemplate(const uint8_t size,
+                                   const uint8_t serialDataPin,
+                                   const uint8_t clockPin,
+                                   const uint8_t latchPin);
+
+  void     setAll(const uint8_t *digitalValues);
+  uint8_t* getAll();
```

Comment:
Looks like every use of this function could also be const.
Of make both versions, one const and one non const.

The const version:
```c++
  const uint8_t* getAll() const;
```

With this being const, the function calling this can also be made const.